modules/audigentRtdProvider.js

```diff
@@ -29,6 +31,9 @@ function addSegmentData(adUnits, data) {
     if (adUnit.hasOwnProperty('bids')) {
       adUnit.bids.forEach(bid => {
         bid.audigent_segments = data;
+        if (segmentMap[bid.bidder] && data[bid.bidder]) {
+          segmentMap[bid.bidder](bid, data[bid.bidder]);
```

Comment:
@anthonylauzon don't you want to add some `try-catch` around this part with appropriate error message, after all, you're calling a user-defined function here?